packages/daimo-api/src/server/pushNotifier.ts

```diff
@@ -229,6 +239,65 @@ export class PushNotifier {
     ];
   }
 
+  private getPushMessagesFromRequests(
+    logs: DaimoRequestV2Status[]
+  ): ExpoPushMessage[] {
+    const messages = [];
+
+    for (const log of logs) {
+      // Only proceed if log is relevant.
+      if (
+        [DaimoRequestState.Pending, DaimoRequestState.Cancelled].includes(
+          log.status
+        )
+      ) {
+        const { tokenSymbol } = chainConfig;
+        const {
+          link: { dollars },
+          metadata,
+        } = log;
```

Comment:
calculate `dollars` as above ^ in this file, using `toFixed(2)`

Reply:
I believe `dollars` here is already formatted in `RequestIndexer`'s `handleRequestCreated`. `getPushMessagesFromNoteOps` also uses `dollars` like this.